backend/src/index.ts

```diff
@@ -16,6 +17,7 @@ async function main() {
       console.log(`GraphIQL available at http://localhost:${CONFIG.PORT}`)
     }
   })
+  startValidateCommunities(Number(CONFIG.FEDERATION_VALIDATE_COMMUNITY_TIMER))
```

Comment:
the Number part should be enforced in `config.index.ts`

Reply:
done